### PR TITLE
Fix multi-encoder ASR integration test

### LINKED ISSUE #1405: Multi encoder integration test is dead but not reported in CI
https://travis-ci.org/espnet/espnet/jobs/620043261#L14985

I'm afraid that this line does not work well.
https://github.com/espnet/espnet/blob/2b9e8476b357eee78c0d5896833eeaeb0d1d5efe/ci/test_integration.sh#L8

### PR DESCRIPTION
**Pull request by ShigekiKarita**
fix #1405

### CODE REVIEW

**Comment by ShigekiKarita** [comment]
Now I confirmed this CI collectly failed by the error in mulenc recipe. See https://travis-ci.org/espnet/espnet/jobs/624865483#L4584 (Thanks @kamo-naoyuki)

File: espnet/asr/pytorch_backend/asr.py [hunk 1]
```diff
@@ -65,6 +65,14 @@
     from itertools import zip_longest as zip_longest
 
 
+def _recur_to(xs, device):
```
Comment (sw005320) [inline espnet/asr/pytorch_backend/asr.py]: what does `recur` stand for?

**Comment by ShigekiKarita** [comment]
Recursive. Do you wanna rename it to _recursive_to?

2019年12月14日(土) 13:13 Shinji Watanabe <notifications@github.com>:

**Comment by sw005320** [comment]
that sounds better for me.

**Comment by ShigekiKarita** [comment]
Done

**Pull request merged**